node/src/Pos/Launcher/Runner.hs

```diff
@@ -218,8 +219,9 @@ runServer mkTransport mkReceiveDelay mkL (OutSpecs wouts) withNode afterNode oq
     stdGen <- liftIO newStdGen
     logInfo $ sformat ("Our verInfo: "%build) ourVerInfo
     node mkTransport mkReceiveDelay mkConnectDelay stdGen bipPacking ourVerInfo defaultNodeEnvironment $ \__node ->
-        NodeAction mkListeners' $ \converse ->
-            let sendActions :: SendActions m
+        NodeAction mkListeners' $ \__converse ->
+            let converse    = withFailureCleanup oq __converse
```

Comment:
I have merely played the game of "seek 'n replace", so I have not enough knowledge of the codebase to really understand myself whether we 2nd call inside `bracket` should be wrapped with `withFailureCleanup` 🤔